### ISSUE TITLE
WCAG: Modal dialog is missing its roles states and name 

### ISSUE THREAD

**Issue by dan-bowerman** [fgpv-vpgf/fgpv-vpgf#3920]
Spawned from #3892.

"When the side panel is open the background is greyed out visually which makes it a modal. It should announce as such and should keep keyboard focus inside it. Add to the dialog container role="dialog" aria-modal="true" aria-labelledby="[id of heading for dialog]""

https://w3c.github.io/aria-practices/examples/dialog-modal/dialog.html
